packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -186,24 +186,37 @@ export const socketIoPath = "socket.io";
  * Get a stripped version of a URL safe for r11s telemetry
  * @returns undefined if no appropriate hostName is provided
  */
-export function getUrlForTelemetry(hostName: string, path?: string): string | undefined {
+export function getUrlForTelemetry(hostName: string, path: string = ""): string | undefined {
 	// Strip off "http://" or "https://"
 	const hostNameMatch = hostName.match(/^(?:https?:\/\/)?([^/]+)/);
 	if (!hostNameMatch) {
 		return undefined;
 	}
 	const strippedHostName = hostNameMatch[1];
 
-	let extractedPath: string | undefined;
-	if (path !== undefined) {
-		// Extract the first portion of the path and explicitly match it to known path names
-		const pathMatch = path.match(/^\/?([^/]+)/);
-		if (pathMatch && [socketIoPath, "repos", "deltas", "documents"].includes(pathMatch[1])) {
-			extractedPath = pathMatch[1];
+	let extractedPath = "";
+	// Extract portions of the path and explicitly match them to known path names
+	for (const portion of path.split("/")) {
+		if (portion !== "") {
+			// eslint-disable-next-line unicorn/prefer-ternary
+			if (
+				[
+					socketIoPath,
+					"repos",
+					"deltas",
+					"documents",
+					"session",
+					"git",
+					"summaries",
+					"latest",
```

Comment:
let's add the following as well:

- document
- commits
- blobs
- refs
- revokeToken
- accesstoken